pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md

```diff
@@ -0,0 +1,128 @@
+    kubectl config use-context <context for first eks cluster>
+    ```
+
+1. Confirm `kubectl` can access the EKS cluster.  
+
+    ```bash
+    kubectl get no
+    ```
+    
+1. Create a service account for Kommander on your EKS cluster.
+
+    ```bash
+    kubectl -n kube-system create serviceaccount kommander-cluster-admin
+    ```
+    
+1. Configure your `kommander-cluster-admin` service account to have `cluster-admin` permissions. Enter the following commands:
+
+    ```yaml
+    cat << EOF | kubectl apply -f -
+    apiVersion: rbac.authorization.k8s.io/v1beta1
+    kind: ClusterRoleBinding
+    metadata:
+      name: kommander-cluster-admin
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: cluster-admin
+    subjects:
+    - kind: ServiceAccount
+      name: kommander-cluster-admin
+      namespace: kube-system
+    EOF
+    ```
+
+1. Create a kubeconfig file compatible with the Kommander UI. Enter these commands to set the following environment variables:
+
+    ```bash
+    export USER_TOKEN_NAME=$(kubectl -n kube-system get serviceaccount kommander-cluster-admin -o=jsonpath='{.secrets[0].name}')
+    export USER_TOKEN_VALUE=$(kubectl -n kube-system get secret/${USER_TOKEN_NAME} -o=go-template='{{.data.token}}' | base64 --decode)
+    export CURRENT_CONTEXT=$(kubectl config current-context)
+    export CURRENT_CLUSTER=$(kubectl config view --raw -o=go-template='{{range .contexts}}{{if eq .name "'''${CURRENT_CONTEXT}'''"}}{{ index .context "cluster" }}{{end}}{{end}}')
+    export CLUSTER_CA=$(kubectl config view --raw -o=go-template='{{range .clusters}}{{if eq .name "'''${CURRENT_CLUSTER}'''"}}{{ index .cluster "certificate-authority-data" }}{{end}}{{ end }}')
+    export CLUSTER_SERVER=$(kubectl config view --raw -o=go-template='{{range .clusters}}{{if eq .name "'''${CURRENT_CLUSTER}'''"}}{{ .cluster.server }}{{end}}{{ end }}')
+    ```
+
+1. Confirm these variables have been set correctly:
+
+    ```bash
+    env | grep CLUSTER
+    ```
+
+1. Create a new kubeconfig file to use in the Kommander UI. Enter the following commands:
+
+    ```bash
+    cat << EOF > kommander-cluster-admin-config
+    apiVersion: v1
+    kind: Config
+    current-context: ${CURRENT_CONTEXT}
+    contexts:
+    - name: ${CURRENT_CONTEXT}
+      context:
+        cluster: ${CURRENT_CONTEXT}
+        user: kommander-cluster-admin
+        namespace: kube-system
+    clusters:
+    - name: ${CURRENT_CONTEXT}
+      cluster:
+        certificate-authority-data: ${CLUSTER_CA}
+        server: ${CLUSTER_SERVER}
+    users:
+    - name: kommander-cluster-admin
+      user:
+        token: ${USER_TOKEN_VALUE}
+    EOF
+    ```
+
+1. Verify the kubeconfig file can access the EKS cluster.
+
+    ```bash
+    kubectl --kubeconfig $(pwd)/kommander-cluster-admin-config get all --all-namespaces
+    ```
+
+1. Copy `kommander-cluster-admin-config` to your clipboard.
+
+    ```bash
+    cat kommander-cluster-admin-config | pbcopy
```

Comment:
this command will only work on MacOS. Need a linux version.

Reply:
There is not a pbcopy for linux. Also, what is done with that file after it's on the clipboard?

Reply:
Would they use xclip?

Reply:
you could just have them run `cat kommander-cluster-admin-config` and copy/paste the yaml file.

you paste the file into the kommander ui. looks like it also supports uploading the file directly.

attaching a screenshot.
![Screen Shot 2020-05-18 at 4 34 12 PM](https://user-images.githubusercontent.com/16764963/82257166-93290480-9925-11ea-9f53-2130d90dec42.png)